mvlearn/embed/gcca.py

```diff
@@ -58,6 +58,10 @@ class GCCA(BaseEmbed):
     tall : boolean, default=False
         Set to true if n_samples > n_features, speeds up SVD
 
+    max_ranks : boolean, default=False
```

Comment:
Is this typically suited for certain scenarios? Can you maybe add something to the Notes section saying what difference this might make in the output?